3.3/.sti/bin/assemble

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+#
+# Default STI assemble script for the python-3.3 image.
+#
+source "../lib/python_context"
```

Comment:
Also, I'd recommend not using -e in shebang and instead add it after sourcing scl_source. That way you can avoid needing to set +e inside .sti/bin/run